guacamole-existing-vnet/azuredeploy.json

```diff
@@ -0,0 +1,236 @@
+{
+  "$schema": "https://schema.management.azure.com/schemas/2015-01-01/deploymentTemplate.json#",
+  "contentVersion": "1.0.0.0",
+  "parameters": {
+    "location": {
```

Comment:
For the location, I'd prefer the user to specify it because it has to be the same location of the existing VNet. However, if you still think resourceGroup().location is a better approach, i'll change it.